training/bert_training_legacy.py

```diff
@@ -0,0 +1,88 @@
+import pandas as pd
+import numpy as np
+import tensorflow as tf
+from utils import bert_text_processor as btp
+from models import bert_cve_classifier as bcvec
+from sklearn.utils import class_weight
+from collections import Counter
+from sklearn.model_selection import train_test_split
+
+SEED = 42
+np.random.seed(SEED)
+tf.set_random_seed(SEED)
+
+
+dataset = pd.read_csv(
+    "/Users/avgupta/GH_complete_labeled_issues_prs - preprocessed.csv",
+    encoding="utf-8",
+    na_filter=False,
+)
+dataset = dataset[dataset.label != 0]
+texts = dataset["description"].tolist()
+labels = dataset["label"].tolist()
+
+print("Before:", Counter(labels))
+labels = [0 if item == 1 else 1 for item in labels]
+print("After:", Counter(labels))
+
+
+train_text, test_text, train_labels, test_labels = train_test_split(
+    texts, labels, test_size=1e-4, random_state=SEED
+)
+print("Before:", len(train_text), len(test_text))
+
+
+# Initialize session
+sess = tf.Session()
+
+# Params for bert model and tokenization
+BERT_PATH = "/Users/avgupta/vm-models-original/model_assets/gokube-phase2/base_bert_tfhub_models/bert_uncased_L12_H768_A12"
```

Comment:
same as above, can we read it from s3?

Reply:
Changed  it to reading from cloud_constants, we download this locally to EMR in bootstrap so don't need read from S3 here.